source/includes/_file_proxy_api.md.erb

```diff
@@ -0,0 +1,78 @@
+# HMDA File Serving
+
+The HMDA Platform serves a number of files including:
+
+1. Documentation
+1. Modified LAR
+1. Institution Register Summaries
+1. Dynamic National Loan-Level Dataset
+1. Snapshot National Loan-Level Dataset
+1. Rate Spread Data
```

Comment:
Below you mention that the `list of files is available on` our page but the files are kind of all over the place. Should we link these for easy access?  
 
Then below, point users back to this list to for links. 
 
 ```suggestion
1. [Documentation](https://ffiec.cfpb.gov/documentation)
1. [Modified LAR](https://ffiec.cfpb.gov/data-publication/modified-lar/)
1. [Institution Register Summaries](https://ffiec.cfpb.gov/filing)
1. [Dynamic National Loan-Level Dataset](https://ffiec.cfpb.gov/data-publication/dynamic-national-loan-level-dataset)
1. [Snapshot National Loan-Level Dataset](https://ffiec.cfpb.gov/data-publication/snapshot-national-loan-level-dataset)
1. Rate Spread Data
```